x/auth/client/cli/tx_sign.go

```diff
@@ -230,64 +228,52 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		// if --signature-only is on, then override --append
-		generateSignatureOnly, _ := cmd.Flags().GetBool(flagSigOnly)
-		multisigAddrStr, _ := cmd.Flags().GetString(flagMultisig)
-
-		from, _ := cmd.Flags().GetString(flags.FlagFrom)
+		printSignatureOnly, _ := f.GetBool(flagSigOnly)
+		multisigAddrStr, _ := f.GetString(flagMultisig)
+		from, _ := f.GetString(flags.FlagFrom)
 		_, fromName, err := client.GetFromFields(txFactory.Keybase(), from, clientCtx.GenerateOnly)
 		if err != nil {
 			return fmt.Errorf("error getting account from keybase: %w", err)
 		}
 
+		overwrite, _ := f.GetBool(flagOverwrite)
 		if multisigAddrStr != "" {
 			var multisigAddr sdk.AccAddress
-
 			multisigAddr, err = sdk.AccAddressFromBech32(multisigAddrStr)
 			if err != nil {
 				return err
 			}
-
 			err = authclient.SignTxWithSignerAddress(
-				txF, clientCtx, multisigAddr, fromName, txBuilder, clientCtx.Offline,
-			)
-			generateSignatureOnly = true
+				txF, clientCtx, multisigAddr, fromName, txBuilder, clientCtx.Offline, overwrite)
+			printSignatureOnly = true
 		} else {
-			flagAppend, _ := cmd.Flags().GetBool(flagAppend)
-			appendSig := flagAppend && !generateSignatureOnly
-			if appendSig {
-				err = authclient.SignTx(txF, clientCtx, clientCtx.GetFromName(), txBuilder, clientCtx.Offline)
-			}
+			err = authclient.SignTx(txF, clientCtx, clientCtx.GetFromName(), txBuilder, clientCtx.Offline, overwrite)
 		}
 		if err != nil {
 			return err
 		}
 
-		aminoJSON, _ := cmd.Flags().GetBool(flagAmino)
-
+		aminoJSON, err := f.GetBool(flagAmino)
 		if err != nil {
 			return err
 		}
 
 		var json []byte
-
```

Comment:
Removing blank lines usually attracts @alexanderbez's ire :)

Jokes aside, I think that having strategically and logically placed blank lines improve readability. Wdyt @robert-zaremba?

Reply:
But the day we enable the `golangci-lint`'s `wsl` linter, this will show up. But it works for me either way.

Reply:
:D